bin/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Prevent jenkins from immediately killing the script when a step fails, allowing us to notify github:
```

Comment:
Is a stub universe the final product of this script?  If so, it should be called something like "make-stub-universe.sh".  "build" implies that it's building spark.  It's sort of building the Spark package, so it kind of makes sense as-is, but a universe is not a package, so not really.